ramda.d.ts

```diff
@@ -1766,8 +1766,8 @@ declare module R {
          *      f(100); //=> true
          *      f(101); //=> false
          */
-        and<T extends (x) => boolean>(fn1: T, fn2: T): T;
-        and<T extends (x) => boolean>(fn1: T): (fn2: T) => T;
+        and<T extends (x) => boolean>(fn1: T, fn2: T: any): T;
+        and<T extends (x) => boolean>(fn1: T): (fn2: T: any) => T;
 
```

Comment:
Type T extends a function with one argument that has not type, thus it should be ...<T extends (x: any) => boolean>(...
Like in your fix for `or`